core/tauri-build/src/lib.rs

```diff
@@ -211,7 +211,9 @@ fn copy_frameworks(dest_dir: &Path, frameworks: &[String]) -> Result<()> {
 // creates a cfg alias if `has_feature` is true.
 // `alias` must be a snake case string.
 fn cfg_alias(alias: &str, has_feature: bool) {
```

Comment:
we pretty much have this function on all crates, can you also update those? just search for `cargo:rustc-cfg`

Reply:
sure thing